runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -114,27 +124,31 @@ private void sortOverlays()
 					.stream()
 					.flatMap(plugin -> plugin.getOverlays().stream()),
 				Stream.of(infoBoxOverlay, tooltipOverlay))
-			.sorted((a, b) ->
-			{
-				if (a.getPosition() != b.getPosition())
-				{
-					// This is so non-dynamic overlays render after dynamic
-					// overlays, which are generally in the scene
-					return a.getPosition().compareTo(b.getPosition());
-				}
-
-				// For dynamic overlays, higher priority means to
-				// draw *later* so it is on top.
-				// For non-dynamic overlays, higher priority means
-				// draw *first* so that they are closer to their
-				// defined position.
-				return a.getPosition() == OverlayPosition.DYNAMIC
-					? a.getPriority().compareTo(b.getPriority())
-					: b.getPriority().compareTo(a.getPriority());
-			})
 			.collect(Collectors.toList()));
```

Comment:
Is a sort call needed after this?

Reply:
Yes, I think so. I forgot to call it, I will update PR.